### ISSUE TITLE
Is the refcoco-trained model available?

### ISSUE THREAD

**Issue by st2yang** [jackroos/VL-BERT#67]
Hello,

It seems that only generic pre-trained models are available, and we have to fine-tune the model on the specific task, right? I am wondering if the refcoco-trained model is available for evaluation?

Thanks,
st2yang

**Comment by jackroos** [comment]
Yes, you need to finetune them on downstream tasks. And I am sorry these models are not available since I didn't store them until now.

**Comment by MI-WENYU** [comment]
> Hello, st2yang
> 
>Can you reproduce the results in the author's paper when fine-tuning the program (for the Refcoco dataset)? When I fine-tuned the test, the accuracy was only about 44 percent
> 
> Thanks

**Comment by MI-WENYU** [comment]
> Hello,
> 
> It seems that only generic pre-trained models are available, and we have to fine-tune the model on the specific task, right? I am wondering if the refcoco-trained model is available for evaluation?
> 
> Thanks, st2yang

Hello, st2yang

Can you reproduce the results in the author's paper when fine-tuning the program (for the Refcoco dataset)? When I fine-tuned the test, the accuracy was only about 44 percent

Thank

**Comment by st2yang** [comment]
@MI-WENYU Yes

**Comment by MI-WENYU** [comment]
> @MI-WENYU Yes
@st2yang 
Does the reproduction process use the original parameters? I have been unable to achieve the results in the paper. Could you please share your fine-tuned model with me? I would be very grateful.
